.travis.yml

```diff
@@ -60,6 +60,7 @@ jobs:
         - &STANDARD_TEST_JOB
             stage: Fast Test
             php: 7.0
+            env: COMPOSER_FLAGS="--prefer-stable"
```

Comment:
why those changes?

Reply:
can we ask for new stable release of vfsstream ?

Reply:
I see they already released it three days ago: https://github.com/bovigo/vfsStream/commit/231c73783ebb7dd9ec77916c10037eff5a2b6efe

I removed related changes in this PR.